python/python/tests/test_async_client.py

```diff
@@ -1208,6 +1208,22 @@ async def test_zpopmin(self, redis_client: TRedisClient):
 
         assert await redis_client.zpopmin("non_exisitng_key") == {}
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_zpopmax(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        members_scores = {"a": 1.0, "b": 2.0, "c": 3.0}
+        assert await redis_client.zadd(key, members_scores=members_scores) == 3
```

Comment:
minor- why are you passing the name of the argument, members_scores=members_scores?